lib/astutils.cpp

```diff
@@ -133,9 +133,74 @@ const Token * astIsVariableComparison(const Token *tok, const std::string &comp,
     return ret;
 }
 
+const Token * getVariableInitExpression(const Variable * var)
```

Comment:
these helper functions are still not static. please add the static keyword for all internal functions.

    static const Token * getVariableInitExpression(const Variable * var)